Wandb-visualization/README.md

```diff
@@ -0,0 +1,100 @@
+# Wandb visualization
+
+> [!NOTE]  
+> If you wish to run the included examples on LUMI, have a look at the [quickstart](../quickstart/README.md) chapter for instructions on how to set up the required environment.
+
+[Weights & Biases](https://docs.wandb.ai/guides) is an AI developer platform consisting of three major components: [Models](https://docs.wandb.ai/guides/models/), [Weave](https://wandb.github.io/weave/) and [Core](https://docs.wandb.ai/guides/core/). These components provide features such as tracking your runs, models and datasets, visualizing data and hyperparameter sweeps.
+
+
+## Initializing Wandb
+
+[W&B QuickStart](https://docs.wandb.ai/quickstart/) Consists of 2 simple steps: registering an account and authenticating with the API key on LUMI.
+
+Registering and getting the API key can be done [here](https://wandb.ai/authorize). After having received the API key, the following commands will authenticate your account on LUMI.
+
+```bash
+export WANDB_API_KEY=<your_api_key>
+pip install wandb
+wandb login
+```
+
+Your login details will be saved in `~/.netrc`, which can be deleted to effectively log out.
+
+## Collecting Logs & Tracking Parameters
+
+[Weights & Biases Models Experiments](https://docs.wandb.ai/guides/track/) provide a simple way to track machine learning experiments with a few lines of code. You can then review the results in an interactive dashboard/
```

Comment:
> You can then review the results in an interactive dashboard/

Maybe specify where this dashboard is (on the wandb website). I though first it is on the LUMI web-app dashboard since this is used in the previous two chapters.